src/device.js

```diff
@@ -77,7 +77,23 @@ var devicePixelRatio = window.originalDevicePixelRatio || window.devicePixelRati
 // @deprecated
 exports.devicePixelRatio = devicePixelRatio;
 
-exports.screen.devicePixelRatio = devicePixelRatio;
+var ctx = document.createElement('Canvas').getContext('2d');
```

Comment:
Never mind, I see these values are getting updated all over the place, it might be able to be cleaned up, I'm skeptical of some of the older code hanging around.